presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/PushAggregationThroughOuterJoin.java

```diff
@@ -269,18 +272,19 @@ private static boolean groupsOnAllColumns(AggregationNode node, List<Symbol> col
         // to symbol references for the new values node.
         NullLiteral nullLiteral = new NullLiteral();
         ImmutableList.Builder<Symbol> nullSymbols = ImmutableList.builder();
-        ImmutableList.Builder<Expression> nullLiterals = ImmutableList.builder();
+        ImmutableList.Builder<RowExpression> nullConstants = ImmutableList.builder();
         ImmutableMap.Builder<Symbol, SymbolReference> sourcesSymbolMappingBuilder = ImmutableMap.builder();
         for (Symbol sourceSymbol : referenceAggregation.getSource().getOutputSymbols()) {
-            nullLiterals.add(nullLiteral);
```

Comment:
This is no longer needed?

Reply:
It's called `nullConstants`. I accidentally moved the logic down. Will move up

Reply:
NVM, I will just use `toRaw` for safety.